src/translator_en.h

```diff
@@ -1039,7 +1039,8 @@ class TranslatorEnglish : public Translator
         "This will result in the following graph:"
         "<p><center><img alt=\"\" src=\"graph_legend."+getDotImageExtension()+"\"></center></p>\n"
         "<p>\n"
-        "The boxes in the above graph have the following meaning:\n"
+        "For the default configuration, "
+        "the boxes in the above graph have the following meaning:\n"
```

Comment:
Does this mean that the "legend" is in the "old" stile even when using "shape=plain" and thus having an inconsistent "legend" with the images? 
The "legend" should follow the current status (there might also be some places already where the color description doesn't fit).

Reply:
Do you mean legend text? For default configuration the legend image is consistent with legend text. For another style configuration legend images are inconsistent with the hard-coded legend text.
What do you suggest?

Reply:
I think that the legend (and its texts) should express the the way the graphs are generated (i.e. colors, shapes and texts).

Reply:
We have contradiction between flexible graph generation rigid text description on many languages.

Hypothetically it is possible to translate arbitrary graph style to description to all languages with help of AI.